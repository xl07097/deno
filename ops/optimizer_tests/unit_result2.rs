pub fn op_set_nodelay(
  state: &mut OpState,
  rid: ResourceId,
  nodelay: bool,
) -> Result<(), AnyError> {
  let resource: Rc<TcpStreamResource> =
    state.resource_table.get::<TcpStreamResource>(rid)?;
  resource.set_nodelay(nodelay)
}
