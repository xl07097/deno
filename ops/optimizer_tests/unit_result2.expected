=== Optimizer Dump ===
returns_result: true
has_ref_opstate: true
has_rc_opstate: false
has_fast_callback_option: false
needs_fast_callback_option: false
fast_result: Some(Void)
fast_parameters: [V8Value, U32, Bool]
transforms: {}
is_async: false
fast_compatible: true
